src/auth/negotiate/kerberos/negotiate_kerberos.h

```diff
@@ -114,7 +114,7 @@ char *gethost_name(void);
 
 #if (HAVE_GSSKRB5_EXTRACT_AUTHZ_DATA_FROM_SEC_CONTEXT || HAVE_GSS_MAP_NAME_TO_ANY) && HAVE_KRB5_PAC
 #define HAVE_PAC_SUPPORT 1
-#define MAX_PAC_GROUP_SIZE 200*60
+#define MAX_PAC_GROUP_SIZE 1024*98
```

Comment:
If possible, please briefly explain these constants, especially since there were [concerns](https://bugs.squid-cache.org/show_bug.cgi?id=5407#c4) about their earlier variant. Please do that in a C++ comment (rather than GitHub comment). For _example_ (I do _not_ know the right terminology, so edit as needed!):

```suggestion
/// the desired maximum number of membership groups (per authenticated user)
/// multiplied by the maximum encoded group entry size
#define MAX_PAC_GROUP_SIZE (1024*98)
```

The above example also adds parenthesis to improve this code safety. I do not insist on that addition or on replacing this macro with a named size_t constant.